source/releasenotes/2024-04-xx-pantheon-page-cache-default-ttl.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Default Cache TTL bumped to one week
+published_date: "2024-04-xx"
+categories: [wordpress, plugins]
+---
+
+With the latest [1.4.0 release](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin, we have updated the default cache TTL to one week. Previously, the default value was `600` (10 minutes). This change is intended to improve the performance of your site by reducing the number of cache clears that are run automatically.
```

Comment:
With the latest [1.4.0 release](https://github.com/pantheon-systems/pantheon-mu-plugin/releases) of the Pantheon MU Plugin, we have updated the default cache TTL to one week. Previously, the default value was 10 minutes. This change is intended to improve the performance of your site by reducing the number of cache clears that are run automatically.

Reply:
We should also note here that the Pantheon Advanced Page Cache plugin auto-clears content, which is why we're increasing the default cache TTL.

Also should add a note that this change does not the TTL for existing sites.